src/Sylius/Component/Product/Model/Product.php

```diff
@@ -26,9 +26,14 @@
  * @author Paweł Jędrzejewski <pawel@sylius.org>
  * @author Gonzalo Vilaseca <gvilaseca@reiss.co.uk>
  */
-class Product extends AbstractTranslatable implements ProductInterface
+class Product implements ProductInterface
 {
-    use SoftDeletableTrait, TimestampableTrait, ToggleableTrait;
+    use SoftDeletableTrait;
+    use TimestampableTrait;
+    use ToggleableTrait;
+    use TranslatableTrait {
+        __construct as translatableConstruct;
```

Comment:
and even more: can't we initialize the `ArrayCollection()` in the class? not in trait?